### ISSUE TITLE
Main Thread Checker warning with Message Center

### ISSUE THREAD

**Issue by brocoo** [urbanairship/ios-library#158]
I've had a few crashes on the app I'm working on that seem to be caused by the Message Center. After investigating the stack trace, it turns out that the SDK is doing some UI changes from a background thread. 
See the warning below returned by Xcode 9 when opening the app from a Push Notification that deep links into the Message Center:

```
=================================================================
Main Thread Checker: UI API called on a background thread: -[UIRefreshControl init]
PID: 306, TID: 17144, Thread name: (none), Queue name: NSManagedObjectContext 0x1c41cca80, QoS: 0
Backtrace:
4   AirshipKit                          0x0000000103ba81f8 -[UADefaultMessageCenterListViewController initWithNibName:bundle:] + 360
5   AirshipKit                          0x0000000103b61a08 -[UADefaultMessageCenterSplitViewController configure] + 128
6   AirshipKit                          0x0000000103b61c2c -[UADefaultMessageCenterSplitViewController initWithNibName:bundle:] + 112
7   AirshipKit                          0x0000000103b487f4 -[UADefaultMessageCenter display:] + 136
8   AirshipKit                          0x0000000103b48c24 -[UADefaultMessageCenter displayMessageForID:animated:] + 60
9   AirshipKit                          0x0000000103bbf0ec -[UADisplayInboxAction displayInboxMessage:situation:] + 636
10  AirshipKit                          0x0000000103bbec74 __63-[UADisplayInboxAction performWithArguments:completionHandler:]_block_invoke + 164
11  AirshipKit                          0x0000000103b776b4 __75-[UAInboxMessageList retrieveMessageListWithSuccessBlock:withFailureBlock:]_block_invoke.69 + 120
12  AirshipKit                          0x0000000103b78c34 __56-[UAInboxMessageList refreshInboxWithCompletionHandler:]_block_invoke + 648
13  AirshipKit                          0x0000000103ba5874 __61-[UAInboxStore fetchMessagesWithPredicate:completionHandler:]_block_invoke + 636
14  CoreData                            0x00000001835e6ea4 <redacted> + 156
15  libdispatch.dylib                   0x0000000105bad45c _dispatch_client_callout + 16
16  libdispatch.dylib                   0x0000000105bbc110 _dispatch_queue_serial_drain + 692
17  libdispatch.dylib                   0x0000000105bb09a4 _dispatch_queue_invoke + 332
18  libdispatch.dylib                   0x0000000105bbd104 _dispatch_root_queue_drain_deferred_wlh + 424
19  libdispatch.dylib                   0x0000000105bc4100 _dispatch_workloop_worker_thread + 652
20  libsystem_pthread.dylib             0x00000001809d2fe0 _pthread_wqthread + 932
21  libsystem_pthread.dylib             0x00000001809d2c30 start_wqthread + 4
```
And this is with the version 8.6.2 of the SDK

**Comment by BrianBatchelder** [comment]
@brocoo - We have identified the issue and will release a patch later today.

**Comment by BrianBatchelder** [comment]
@brocoo - fixed in [8.6.3](https://github.com/urbanairship/ios-library/tree/8.6.3)
